templates/hooks/ipset/sdk_read_one_post_set_output.go.tpl

```diff
@@ -0,0 +1,8 @@
+	if resp.IPSet != nil {
+		if resp.IPSet.Description != nil {
+			ko.Spec.Description = resp.IPSet.Description
+		}
+		if resp.IPSet.Addresses != nil {
+			ko.Spec.Addresses = resp.IPSet.Addresses
+		}
+	}
```

Comment:
Instead, you can use https://github.com/aws-controllers-k8s/code-generator/blob/main/pkg/config/operation.go#L38-L40 to target `resp.IPSet`